frontend/src/metabase/core/components/TabButton/TabButton.unit.spec.tsx

```diff
@@ -85,4 +85,21 @@ describe("TabButton", () => {
     expect(onRename).toHaveBeenCalledWith(newLabel);
     expect(await screen.findByDisplayValue(newLabel)).toBeInTheDocument();
   });
+
+  it("should limit the length to 75chars", async () => {
+    const { onRename } = setup();
+
+    userEvent.click(getIcon("chevrondown"));
+    (await screen.findByRole("option", { name: "Rename" })).click();
+
+    const newLabel = Array.from({ length: 100 }).fill("a").join("");
```

Comment:
probably a more elegant way is 

```suggestion
    const newLabel = "a".repeat(100);
```